Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/Util/DatabaseIndexHelper.swift

```diff
@@ -13,8 +13,8 @@ public class DatabaseIndexHelper {
         DispatchQueue.global(qos: .userInitiated).async { [weak self] in
             self?.queue.inTransaction { db, rollback in
                 do {
+                    try db.executeUpdate("DROP INDEX IF EXISTS episode_archived;", values: nil)
```

Comment:
@bjtitus just to let you know that I'm removing the `archived` index. It's a lot quality index and looks like is causing issues for some users.